CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 1.2.0-rc1 (2020-11-06)
+* Preview support for daisy chaining IoT Edge devices in gateway configuration, to allow creation of hierarchies of IoT Edge devices.
```

Comment:
**nesting** instead of **daisy chaining**? this is what we have used in the docs